domain_test.go

```diff
@@ -38,21 +38,158 @@ func TestDomainCreate(t *testing.T) {
 	autopilot.Equals(t, "Z2lkOi8vb3BzbGV2ZWwvRW50aXR5T2JqZWN0LzMw", string(result.Id))
 }
 
+func TestDomainGetSystems(t *testing.T) {
+	// Arrange
+	requests := []TestRequest{
+		{`{"query": "query DomainChildSystemsList($after:String!$domain:IdentifierInput!$first:Int!){account{domain(input: $domain){childSystems(after: $after, first: $first){nodes{id,aliases,name,description,htmlUrl,owner{... on Group{alias,id},... on Team{alias,id}},parent{id,aliases,name,description,htmlUrl,owner{... on Group{alias,id},... on Team{alias,id}}}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}}}}",
+			"variables": {
+				{{ template "first_page_variables" }},
+				"domain": {
+					"id": "Z2lkOi8vb3BzbGV2ZWwvRW50aXR5T2JqZWN0LzUx"
+				}
+			}
+			}`,
+			`{
+				"data": {
+					"account": {
+						"domain": {
+							"childSystems": {
+								"nodes": [
+									{{ template "system1_response" }},
+									{{ template "system2_response" }}
+								],
+								{{ template "pagination_initial_pageInfo_response" }},
+								"totalCount": 2
```

Comment:
`totalCount` can get removed - we should no longer be requesting this data in the response